05week/spaceTravelToMars.js

```diff
@@ -9,6 +9,36 @@ let jobTypes = {
   programmer: 'Any Ship!'
 };
 
+class CrewMember {
+  constructor(name, job, specialSkill){
+    this.name = name;
+    this.job = job;
+    this.specialSkill = specialSkill;
+    this.ship = null;
+  }
+  enterShip(ship)  {
+    this.ship = ship;
+    ship.crew.push(this);
```

Comment:
You're pushing the entire class into crew array.

Reply:
Changed the variable to "shipToBeEntered" to be more clear. My tests pass. 
<img width="580" alt="screen shot 2018-12-10 at 8 14 53 pm" src="https://user-images.githubusercontent.com/41026694/49773926-a9567e00-fcb8-11e8-9bb0-d3e028db1a00.png"> @ericbarrs